leftist.ml

```diff
@@ -0,0 +1,69 @@
+(*  Autor kodu: Przemysław Podleśny *)
+(*  Code reviewer: Jakub Nowak      *)
+
+(*  Korzeń kolejki priorytetowej - albo drzewo puste,
+    albo rekord reprezentujący węzeł drzewa             *)
```

Comment:
Labowiec kazał opisywać wszystkie pola rekordów.
Ja wiem, że to głupie, bo wynika z nazw pól, ale tak samo głupie jest pisanie komentarza do każdej funkcji. xD